app/cdap/components/GridTable/components/DirectiveComponents/Calculate/index.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useState, useEffect, Fragment } from 'react';
+import { CALCULATE_OPTIONS } from '../../NestedMenu/menuOptions/calculateOptions';
+import FormInputFieldComponent from '../InputComponents/FormInputFieldComponent';
+import { FormGroup, FormHelperText, Box, Typography } from '@material-ui/core';
+import { useStyles } from '../styles';
+import InputCheckbox from '../InputComponents/InputCheckbox';
+import T from 'i18n-react';
+import LabelComponent from '../InputComponents/LabelInputComponent';
+import { ICalculateProps } from './types';
+
+export default function({
+  functionName,
+  setDirectiveComponentsValue,
+  directiveComponentValues,
```

Comment:
rename variables, functions or anything related to transformations with something specific to transformations